packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -163,6 +180,16 @@ export default function MongoNav({
     });
   }
 
+  function fetchProductionData(body: PostBodyInterface) {
```

Comment:
Fairly sure you could figure out a way to merge this and the get function together. So all fetching is in one location. Don't think it'd add that much complexity either.

Reply:
Related to that we should test that the proper behavior occurs too. 

Ideally with new functionality we also add tests to add coverage there.

Reply:
Something like:

```fetchProductionData(body) {
  if(body) { POST }
  else { GET } 
}```

Reply:
Ideally we only use `fetch` function once and change the args we send to it.

Reply:
👍